### PR TITLE
Add bastion host module

### PR DESCRIPTION
**Pull request by rileykarson**
Implements https://github.com/gruntwork-io/terraform-google-network/pull/14

### CODE REVIEW

**Comment by rileykarson** [comment]
Do one of you mind setting `GOOGLE_IDENTITY_EMAIL` to the test runner? We applied the same change in https://github.com/gruntwork-io/terratest.

**Comment by autero1** [comment]
Done. I assumed the value should be the circleci terratest SA email...

**Comment by autero1** [comment]
Kicked off tests as well and they passed

File: examples/bastion-host/main.tf [hunk 1]
```diff
@@ -1 +1,47 @@
+module "management_network" {
+  # When using these modules in your own templates, you will need to use a Git URL with a ref attribute that pins you
+  # to a specific version of the modules, such as the following example:
+  # source = "git::git@github.com:gruntwork-io/terraform-google-network.git//modules/vpc-network?ref=v0.1.0"
```
Comment (yorinasub17) [inline examples/bastion-host/main.tf]: Should this match the ref tag below @ v0.1.1?

File: modules/bastion-host/main.tf [hunk 2]
```diff
@@ -1 +1,24 @@
+resource "google_compute_instance" "bastion_host" {
```
Comment (yorinasub17) [inline modules/bastion-host/main.tf]: Reminder to add section header docs

File: examples/bastion-host/main.tf (hunk 1 above)
Comment (rileykarson) [reply examples/bastion-host/main.tf]: Bumping to 0.1.2 since I'll make a new release including this.

**Pull request merged**